templates/infra/mysql.tmpl.yaml

```diff
@@ -46,26 +42,47 @@ spec:
           initialDelaySeconds: 5
           periodSeconds: 2
           timeoutSeconds: 1
+        volumeMounts:
+        - name: mysql-data
+          mountPath: /var/lib/mysql
```

Comment:
I'm pretty sure you need a "subPath" here or the mysql container will get really confused at start-up when it sees the lost+found directory. That was my experience in GKE. eg I needed:

          # subPath required so volume's lost+found
          # directory doesn't break mysql's init
          subPath: mysql_data

Reply:
I wasn't aware of this issue. Thanks.